projects/common.py

```diff
@@ -27,11 +27,17 @@ def install_packages(packages, proxy=''):
         sudo('yum -y install {0}'.format(' '.join(packages)), pty=False)
 
 
+def update_packages(packages, proxy=''):
+    with shell_env(http_proxy=proxy, ftp_proxy=proxy):
+        sudo('yum -y update {0}'.format(' '.join(packages)), pty=False)
```

Comment:
Not a big deal, but `install` and `update` really look the same so they should be unified, maybe something like:

    def install_packages(packages):
        yum_command('install', packages)

    def update_packages(packages):
        yum_command('update', packages)

    def yum_command(command, packages):
        ...


The reason for having `install_packages` and `update_packages` is that they encapsulate the commands and in the future we'll be able to switch method used to install/update (yum, dnf, whatever...).